cristal-integration-rest-server/src/main/java/org/xwiki/rest/internal/resources/CristalResourceImpl.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.rest.internal.resources;
```

Comment:
Wrong page name, should be `org.xwiki.contrib`

> A generic maven groupId: org.xwiki.contrib (or org.xwiki.contrib.<module name> if the project has several modules). That's until the project reaches a certain size and visibility, in which case it can have its own maven group id.

From https://contrib.xwiki.org/xwiki/bin/view/Main/#HHostingtools

Reply:
Fixed, see https://github.com/xwiki-contrib/cristal-integration/pull/1/files#diff-d0d7a833e02f9cca2a0bbb98969cef085b3ed7a5477701476f7493b71a4cdd67R20